src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/BeneficialOwnerCorporateValidatorTest.java

```diff
@@ -34,7 +35,7 @@ class BeneficialOwnerCorporateValidatorTest {
 
     @BeforeEach
     public void init() {
-        beneficialOwnerCorporateValidator = new BeneficialOwnerCorporateValidator(new AddressDtoValidator());
+        beneficialOwnerCorporateValidator = new BeneficialOwnerCorporateValidator(new AddressDtoValidator(new DataSanitisation()));
```

Comment:
Note that because you're creating an actual instance of the new sanitiser class and not using a mock this 'unit test' is now essentially testing the behaviour of the BeneficialOwnerCorporateValidator code AND the DataSanitisation code.

Would be better to mock it, I think, but up to you.

Reply:
Added inject mocks to addresses and mocked data sanitisation where needed.

Reply:
Thanks @mattch1 .